app/components/k8s/new-or-edit-resource/template.hbs

```diff
@@ -1,10 +1,17 @@
 <section class="header">
-  <h1>{{if editing 'Edit' 'Add'}} {{label}}</h1>
+  <h1>
+  {{~#if editing}}
```

Comment:
Tildes are fine here but `{{t (if editing 'k8s.newOrEditResource.edit' 'k8s.newOrEditResource.add')}} {{label}}` would make it easy to control whitespace.